common/baselines/librispeech/ls960/gmm/baseline_args.py

```diff
@@ -0,0 +1,467 @@
+                "fft_options": None,
+            }
+        },
+    }
+
+    scorer_args = {"sctk_binary_path": SCTK_BINARY_PATH}
+
+    return util.RasrInitArgs(
+        costa_args=costa_args,
+        am_args=am_args,
+        feature_extraction_args=feature_extraction_args,
+        scorer_args=scorer_args,
+    )
+
+
+def get_monophone_args(
+        feature_flow: str = "mfcc+deriv+norm"
+) -> util.GmmMonophoneArgs:
+
+    linear_alignment_args = {
+        "minimum_segment_length": 0,
+        "maximum_segment_length": 6000,
+        "iterations": 5,
+        "penalty": 0,
+        "minimum_speech_proportion": 0.7,
+        "save_alignment": False,
+        "keep_accumulators": False,
+        "extra_merge_args": None,
+        "extra_config": None,
+        "extra_post_config": None,
+    }
+
+    monophone_training_args = {
+        "name": "mono",
+        "feature_flow": feature_flow,
+        "feature_energy_flow_key": "energy," + feature_flow,
+        "align_iter": 75,
+        "splits": 10,
+        "accs_per_split": 2,
+        "dump_alignment_score_report": True,
+    }
+
+    monophone_recognition_args = {
+        # GmmSystem.recognition() args:
+        "iters": [10],
+        "lm_scales": [10],
+        "optimize_am_lm_scale": True,
+        # meta.System.recog() args:
+        "feature_flow": feature_flow,
+        "pronunciation_scales": [1.0],
+        "lm_lookahead": True,
+        "lookahead_options": None,
+        "create_lattice": True,
+        "eval_single_best": True,
+        "eval_best_in_lattice": True,
+        "search_parameters": {
+            "beam-pruning": 15.0,
+            "beam-pruning-limit": 100000,
+            "word-end-pruning": 0.5,
+            "word-end-pruning-limit": 10000,
+        },
+        "parallelize_conversion": False,
+        "lattice_to_ctm_kwargs": {
+            "fill_empty_segments": False,
+            "best_path_algo": "bellman-ford",
+        },
+        "rtf": 20,
+        "mem": 4,
+        "use_gpu": False,
+    }
+
+    return util.GmmMonophoneArgs(
+        linear_alignment_args, monophone_training_args, monophone_recognition_args
+    )
+
+
+def get_cart_args(
+    use_stress_marker: bool = False,
+    max_leaves: int = 12001,
+    min_obs: int = 1000,
+    hmm_states: int = 3,
+    feature_flow: str = "mfcc+deriv+norm",
+    add_unknown: bool = False,
+) -> util.GmmCartArgs:
+    """
+
+    :param use_stress_marker: use ARPAbet stress marker, please also check for correct lexicon then
+    :param max_leaves: total number of final CART states
+    :param min_obs: minimum observation count needed for a label
+    :param hmm_states: number of states per label for the HMM-topology
+    :param feature_flow:
+    :param add_unknown: set to true if an unknown phoneme exists
+    """
+
+    CartQuestions = (
+        CartQuestionsWithStress if use_stress_marker else CartQuestionsWithoutStress
+    )
+
+    cart_questions_class = CartQuestions(
```

Comment:
There was a [pull request](https://github.com/rwth-i6/i6_experiments/pull/90) where the parameter `n_phones` was introduced to give the felxibility of having also diphone CART, IMO, this should be explicitely set here, so that the user see the possibility.

Reply:
Got it, thanks!